app/presenters/notifications/setup/scheduled-notifications.presenter.js

```diff
@@ -24,17 +24,18 @@ const { transformStringOfLicencesToArray } = require('../../../lib/general.lib.j
  * @param {object} returnsPeriod - the return period including the endDate, startDate and dueDate
  * @param {string} referenceCode - the unique code used to group the notifications in notify
  * @param {string} journey - the journey should be either "reminders" or "invitations"
+ * @param {string} eventId - the event id to link all the notifications to an event
  *
  * @returns {object[]} - the recipients transformed into scheduled notifications
  */
-function go(recipients, returnsPeriod, referenceCode, journey) {
+function go(recipients, returnsPeriod, referenceCode, journey, eventId) {
```

Comment:
This will need to be updated when - https://github.com/DEFRA/water-abstraction-system/pull/1791/files is merged.